src/library/basetracktablemodel.cpp

```diff
@@ -26,6 +27,9 @@ const mixxx::Logger kLogger("BaseTrackTableModel");
 
 const QString kEmptyString = QStringLiteral("");
 
+constexpr double kRelHeightOfCoverartTooltip =
```

Comment:
- Please use "ToolTip" consistent with Qt's terminology (for all naming)
- Rel -> Relative

Reply:
Done!